tests/Unit/NumericalAlgorithms/SphericalHarmonics/Python/Test_Spherepack.py

```diff
@@ -0,0 +1,75 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+
+import random
+import unittest
+
+import numpy as np
+import numpy.testing as npt
+from scipy.special import sph_harm
+
+from spectre.SphericalHarmonics import Spherepack, SpherepackIterator
+
+
+def convert_coef_to_spherepack(complex_coef, l, m):
+    sign = 1.0 if m % 2 == 0 else -1.0
+    part = np.real(complex_coef) if m >= 0 else np.imag(complex_coef)
+    return 0.5 * sign * np.sqrt(2.0 / np.pi) * part
```

Comment:
Is this conversion documented somewhere? If so, could you link to that docs here?

Reply:
yes, I suspect I am counting double by just taking the real part of the complex harmonic. I will check that and update.

Reply:
I was indeed and corrected it in a fixup